ocaml-eglot-util.el

```diff
@@ -16,11 +16,19 @@
 
 ;;; Code:
 
+(require 'json)
 (require 'eglot)
 (require 'cl-lib)
 
 ;; Generic util
 
+(defun ocaml-eglot-util--goto-char (target)
+  "Goto the point TARGET."
+  (when (or (< target (point-min))
+            (> target (point-max)))
+    (widen))
+  (goto-char target))
```

Comment:
What does widen do ? Add columns / lines to the buffer ?
It was wrong before to just call `goto-char` ?

Reply:
TIL that one can "narrow" the buffer  in emacs: https://www.gnu.org/software/emacs/manual/html_node/eintr/Narrowing-_0026-Widening.html